dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -983,6 +983,47 @@
         #     variable: true
         #     variableDocument: true
 
+      # Camunda Exporter ----------
+      # An example configuration for the camunda exporter:
+      #
+      # These setting can also be overridden using the environment variables "ZEEBE_BROKER_EXPORTERS_CAMUNDAEXPORTER_..."
+      #
+      # camundaExporter:
+        # className: io.camunda.exporter.CamundaExporter
+        #
+        # args:
+        #   connect:
+        #     type: elasticsearch
+        #     url: http://localhost:9200
+        #     clusterName: elasticsearch
+        #     dateFormat: yyyy-MM-dd'T'HH:mm:ss.SSSZZ
+        #     socketTimeout: 1000
+        #     connectTimeout: 1000
+        #     username: elastic
+        #     password: changeme
+        #     security:
+        #       enabled: false
+        #       certificatePath: /path/to/certificate
+        #       verifyHostname: true
+        #       selfSigned: false
+        #
+        #   bulk:
+        #     delay: 5
+        #     size: 1000
+        #
+        #   index:
+        #     prefix:
+        #     numberOfShards: 3
+        #     numberOfReplicas: 0
+        #     variableSizeThreshold: 8191
+        #
+        #   retention:
+        #     enabled: false
+        #     minimumAge: 30d
+        #     policyName: camunda-retention-policy
+        #
+        #   createSchema: true
+
```

Comment:
❓ In the Opensearch exporter we have a `aws` section do you know the use case for such?

https://github.com/camunda/camunda/pull/23754/files#diff-6783f1e71cace45bffaa5446fc6bde1f63deb6ae0ff99fd28eeae7e659881253R936-L939

Do we need this?

Reply:
right now in the `search-client-connect` they initialize the open search connector with a `DefaultCredentialsProvider.create()`.
We could extend our configs to provide custom aws configurations and provide our own `AwsCredentialsProvider` too. 
I just followed their example in `OpenSearchSearchClientConfiguration` for simplicity for now. Yet, I'm not sure if the default credential provider should always work out of the box.